pkg/config/config.go

```diff
@@ -36,10 +36,7 @@ func InitConfig() {
 
 	// Handle errors reading the config file
 	if err != nil {
-		klog.Fatalf("could not parse/read config file: %s", err)
-
-		// This is superfluous, but it clearly states what the expected exit code
```

Comment:
Do you want to keep this for that specific error code?

Reply:
Nope I missed it :laughing:

Reply:
I think this is the code for a "Fatal" failure